### ISSUE TITLE
callbacks for seal completion

### ISSUE THREAD

**Issue by whyrusleeping** [filecoin-project/rust-fil-sector-builder#21]
### Description
It would be great if we could get a callback into go land for when the sealing finishes. That should be possible, right?

**Comment by whyrusleeping** [comment]
Expected go code interface:
```go
func SectorBuilderRegisterCallback(cb func(*SectorSealingStatus)) {}
```

This would then trigger for every change to a sectors seal status

**Comment by laser** [comment]
> This would then trigger for every change to a sectors seal status

Is your proposed function missing a sector id parameter? Does `cb` get called once per status change per sector which the underlying sector builder manages?

**Comment by whyrusleeping** [comment]
@laser Yeah, i'm thinking it just fires for everything. Any changes to any sectors (including when sectors get added)
